### PR TITLE
Language in the URL and enhancements

### PR DESCRIPTION
**Pull request by SantiagoDePolonia**
PR introduces language in the URL and detecting language based on browser settings. It also contains some small improvements.

In the future I might create proposition of separating presentation and logic in React.
Also I might to treat all code with prettier to make it more standardized in one of the next PR's.

Regarding changes in PHP files - it might be challenging to change back-end for me to support other languages.

### CODE REVIEW

**Comment by clash82** [comment]
Hey @SantiagoDePolonia, nice work here - thank you for your contribution to this project.

Unfortunately, when I try to build docker on your branch I'm getting this:

```
Step 3/11 : COPY .docker/composer_installer.sh /tmp/composer_installer.sh
 ---> 94fb037f2513
Step 4/11 : RUN sh /tmp/composer_installer.sh
 ---> Running in 7e1e61878a68
: not founder_installer.sh: 2: 
: not founder_installer.sh: 6: 
/tmp/composer_installer.sh: 20: Syntax error: end of file unexpected (expecting "then")
The command '/bin/sh -c sh /tmp/composer_installer.sh' returned a non-zero code: 2
```

I'm running this on linux. Is it working on your OSX?

**Comment by clash82** [comment]
> it might be challenging to change back-end for me to support other languages.

Leave it to me, I will implement missing backend part.

File: assets/consts.js
Comment (clash82) [inline assets/consts.js:3]: Since English is the default language now then maybe we should replace this with `en_kjv`?

Also I was thinking that maybe we could use some default settings depending on the selected language? wdyt?

File: docs/install.md
Comment (clash82) [inline docs/install.md:20]: I've created `Makefile` to simplify all of those steps below. Just type `make start` in the root of our project. If you want to login type `make login`. Refer to `Makefile` included in this repo, pls.

Comment (clash82) [reply docs/install.md:20]: Btw, you can type `make` to obtain a list of all implemented commands.

**Comment by SantiagoDePolonia** [comment]
> I'm running this on linux. Is it working on your OSX?

It looks like git changed `\n` for `\r\n` and it breaks the script. I think that now it's fixed. I modified .gitattributes file.

File: docs/install.md
Comment (SantiagoDePolonia) [reply docs/install.md:20]: Right. Good job! That's something what I missed. It looks like the bin/make.bat might be redundant.

Comment (SantiagoDePolonia) [reply docs/install.md:20]: > Btw, you can type `make` to obtain a list of all implemented commands.

Yes, indeed. I prefer to use Makefile for things like those because it's OS agnostic.

Comment (clash82) [reply docs/install.md:20]: Yeah, I was thinking about removing it, but it still works for Windows users. I will probably move it somewhere else and describe it better in docs.

**Comment by clash82** [comment]
> It looks like git changed \n for \r\n and it breaks the script. I think that now it's fixed. I modified .gitattributes file.

You're right, now it works.

**Comment by clash82** [comment]
@SantiagoDePolonia check latest commit in master. I added support for languages on the backend.

From now on all API requests should be prefixed with language code, eg. `/api/pl/b/book`. Currently only Polish language is supported, English file will be added shortly.

File: assets/consts.js
Comment (clash82) [reply assets/consts.js:3]: This is actually the only question which is holding me before merging your PR.

Comment (SantiagoDePolonia) [reply assets/consts.js:3]: > Since English is the default language now then maybe we should replace this with en_kjv?
It works slightly differently. The language is taken from the browser setting. If there is no language specified in the browser than the English is selected. At least this behaviour I wanted to implement.

> Also I was thinking that maybe we could use some default settings depending on the selected language? wdyt?

Sounds like an interesting feature to implement in the future.

Comment (clash82) [reply assets/consts.js:3]: Cool, thanks for clarification :)

**Review by clash82 — APPROVED** [review]

**Pull request merged**
